frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -761,11 +759,9 @@ const plugin: Plugin<ConsumedExtensions> = [
     type: 'AddAction',
     properties: {
       id: 'helm',
-      url: '/catalog?kind=%5B"HelmChart"%5D',
-      // t('devconsole~Helm Chart')
+      url: '/catalog?catalogType=HelmChart',
       label: '%devconsole~Helm Chart%',
-      // t('devconsole~Browse the catalog to discover and install Helm Charts')
```

Comment:
You cannot delete these comments.

Reply:
This might have been a mistake while fixing merge conflicts.